### DISCUSSION TITLE
Opsgenie Notifcations with Specific Priority

### DISCUSSION THREAD

**Discussion by phillf** [containrrr/watchtower#1893] [Q&A]
I have set up Watchtower to send notifications to Opsgenie but what I am struggling with is having notifications bound to specific Priorities. (eg. P5 - Informational = info, P1 - Critical = panic or fatal, P2 -High = error).

As long as priority is getting passed to Opsgenie then I can configure the alert filters on that end.

How can I set this up?

**Comment by piksel** [comment]
I'm not exactly sure what you mean, but you can add a priority to the notification URL, like `&priority=P2`.

See the Shoutrrr docs for OpsGenie for more information:
https://containrrr.dev/shoutrrr/v0.8/services/opsgenie/

**Reply by phillf** [reply]
Sorry my original post was really that clear.  I know about the appending priority via the URL.

How can I have watchtower differentiate between the different message types when sending alerts to Opsgenie? Right now every message that goes to Opsgenie is a P3.

I want to be able to configure Opsgenie to close an alert if it's a P5 - Informational.

**Reply by piksel** [reply]
Oh, I see. Mapping log levels to priority?

I think this is possible to do in the Shoutrrr API itself, but I don't think it's implemented for OpsGenie. Create a feature request over in https://github.com/containrrr/shoutrrr and I can take a look.

**Reply by phillf** [reply]
Semi-related to this.... is there documentation out there as to what scenarios correlate to the different log levels?

**Reply by phillf** [reply]
(Correlating discussion with issue in shoutrr repo)
containrrr/shoutrrr#425
